spec/system/admin/products_v3/products_spec.rb

```diff
@@ -14,6 +14,10 @@
     login_as user
   end
 
+  let(:producer_search_selector) { 'input[placeholder="Search for producers"]' }
```

Comment:
This spec file if getting huge, how about creating an issue for splitting this into different files based on related specs? i.e products_v3_{feature_name}_specs.rb?

Reply:
I was thinking the same! Better to arrange in a folder too.
Maybe some top-level items can be grouped together. Using CRUD terms for inspiration:
* `products_v3/index_specs.rb` (sorting, pagination and search). Or "view_specs" perhaps.
* `products_v3/update_specs.rb` (updating, edit image) 
* `products_v3/actions_specs.rb` (edit, clone, delete) 

What do you think of that?

Reply:
Exactly David. Folder structure would help us arrange in a better way. Looks good to me 🙌🏻